pkgs/development/tools/build-managers/dub/default.nix

```diff
@@ -4,18 +4,34 @@ let
 
   dubBuild = stdenv.mkDerivation rec {
     name = "dubBuild-${version}";
-    version = "1.6.0";
+    version = "1.7.1";
 
     enableParallelBuilding = true;
 
     src = fetchFromGitHub {
       owner = "dlang";
       repo = "dub";
       rev = "v${version}";
-      sha256 = "1xjr5pp263lbcd4harxy1ybh7q0kzj9iyy63ji6pn66fizrgm7zk";
+      sha256 = "09bcc9bq2z1rbm8sdip1l81y5p8q13r30k02lzifyasiplrnpvlv";
     };
 
     postPatch = ''
+      # TODO Remove with next release which contains https://github.com/dlang/dub/pull/1354
```

Comment:
Why not just apply https://patch-diff.githubusercontent.com/raw/dlang/dub/pull/1354.patch in `patches`?

Reply:
You are right, it's nicer and easier this way.
Pushed the change.